src/main/scala/org/broadinstitute/dsde/firecloud/webservice/UserApiService.scala

```diff
@@ -83,43 +83,43 @@ trait UserApiService extends HttpService with PerRequestCreator with FireCloudRe
         authorizationHeader match {
           // no Authorization header; the user must be unauthorized
           case None =>
-            respondWithErrorReport(Unauthorized, Unauthorized.defaultMessage, requestContext)
-          // browser sent Authorization header; try to query rawls for user status
+            respondWithErrorReport(Unauthorized, "No authorization header in request.", requestContext)
+          // browser sent Authorization header; try to query Sam for user status
```

Comment:
My basis for the Rawls -> Sam renames is that the request is a `Get(UserApiService.samRegisterUserURL)`

Reply:
Commenting here for my own learning .... this is true. Following the URL, it is actually making a call to `Sam` and not `Rawls`. It looks like we still have a bit of a mix between where user info lives wrt Sam vs Rawls.

Reply:
it used to be Rawls, and when it moved to Sam nobody updated the code comments.